src/main/scala/com/codacy/client/bitbucket/v2/service/HookServices.scala

```diff
@@ -32,18 +34,22 @@ class HookServices(client: BitbucketClient) {
       hookUrl: String,
       events: Set[String]
   ): RequestResponse[v2.Webhook] = {
-    val servicesUrl = getServicesUrl(author, repo)
+    val servicesUrl = generateHooksUrl(author, repo)
+    val encodedUuid = URLEncoder.encode(uuid, "UTF-8")
     val payload = Json.obj("active" -> active, "description" -> description, "url" -> hookUrl, "events" -> events)
-    client.putJson(Request(s"$servicesUrl/$uuid", classOf[v2.Webhook]), payload)
+    client.putJson(Request(s"$servicesUrl/$encodedUuid", classOf[v2.Webhook]), payload)
   }
 
   def delete(author: String, repo: String, uuid: String): RequestResponse[Boolean] = {
-    val servicesUrl = getServicesUrl(author, repo)
-    client.delete(s"$servicesUrl/$uuid")
+    val servicesUrl = generateHooksUrl(author, repo)
+    val encodedUuid = URLEncoder.encode(uuid, "UTF-8")
+    client.delete(s"$servicesUrl/$encodedUuid")
   }
 
-  private[this] lazy val BASE_URL: String = "https://api.bitbucket.org/2.0/repositories"
-
-  private[this] def getServicesUrl(owner: String, repo: String) = s"$BASE_URL/$owner/$repo/hooks"
+  private def generateHooksUrl(owner: String, repo: String): String = {
```

Comment:
Codacy found an issue: [generateHooksUrl should have a smaller scope by using private[this]](https://app.codacy.com/gh/codacy/bitbucket-scala-client/pullRequest?prid=5907044)